File: VB/Demos/05Generics/Runner.vb
Namespace Generics
  Module Runner
    Sub RunMe()
      Dim set1 = New WebSettings With {
        .VDir = "c:\temp\test"
      }

      Save(set1, "c:\temp\demo2.dat")

      Dim set2 = DirectCast(Load("c:\temp\demo2.dat"), WebSettings)

      Console.WriteLine(set2.VDir)
    End Sub
  End Module
End Namespace

File: CS/Demos/05Generics/Runner.cs
namespace Demos.Generics;

static class Runner
{
  public static void RunMe()
  {
    var set1 = new WebSettings
    {
      VDir = "c:\\temp\\test"
    };

    SettingsUtils.Save(set1, "c:\\temp\\demo2.dat");

    var set2 = (WebSettings)SettingsUtils.Load("c:\\temp\\demo2.dat");

    Console.WriteLine(set2.VDir);
  }
}
